sql/catalyst/src/main/scala/org/apache/spark/sql/sources/filters.scala

```diff
@@ -29,7 +29,7 @@ import org.apache.spark.annotation.{Evolving, Stable}
  * @since 1.3.0
  */
 @Stable
-abstract class Filter {
+sealed abstract class Filter {
```

Comment:
Hi, @MaxGekk . Please file a JIRA for this.

Reply:
done

Reply:
Thanks.